### ISSUE TITLE
Disable EFUSE option causing compiler error (IDFGH-1477)

### ISSUE THREAD

**Issue by KaeLL** [espressif/esp-idf#3746]
If you enable `CONFIG_ESP32_DISABLE_BASIC_ROM_CONSOLE`, on the current state of the master branch, you get
```
/home/nvmd/Documents/VSCode/esp-idf/components/esp32/cpu_start.c: In function 'start_cpu0_default':
/home/nvmd/Documents/VSCode/esp-idf/components/esp32/cpu_start.c:361:5: error: implicit declaration of function 'esp_efuse_disable_basic_rom_console' [-Werror=implicit-function-declaration]
     esp_efuse_disable_basic_rom_console();
```

One `#include` away from paradise :P

**Comment by KonstantinKondrashov** [comment]
Hi @KaeLL, thanks for your report, it was fixed recently by [commit 09ee721](https://github.com/espressif/esp-idf/commit/09ee7219322c127aaa102a4809d8f29e9bf67cad)

**Comment by Alvin1Zhang** [comment]
@KaeLL Thanks for reporting the issue. Feel free to reopen the ticket if the issue still exists. Thanks.
